src/campaigns/services/campaigns.service.ts

```diff
@@ -235,6 +244,26 @@ export class CampaignsService {
     return true
   }
 
+  async findSlug(user: User, suffix?: string) {
```

Comment:
Let's call this `findBySlug`.. this reads like we're finding a slug, when in fact, we're finding a campaign, by it's slug

Reply:
Oh.. duh.. ok.. total 🧠 💨  on my behalf.. 👍